<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralWidget{
border-image: url(:/png/main/bg_main.png);
}

QLabel{
font-size: 16pt;
}

#mainLab{
color: rgb(255, 255, 255);
font-size: 18pt;
}

#timeLab{
color: rgb(255, 255, 255);
font-size: 16pt;
}

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,14">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>42</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="logoLab">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/png/main/logo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="mainLab">
         <property name="minimumSize">
          <size>
           <width>414</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>主界面</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLab">
         <property name="minimumSize">
          <size>
           <width>413</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bodyWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,6,1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>80</number>
         </property>
         <property name="leftMargin">
          <number>240</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>240</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="generalBtn" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#generalBtn{
border-image: url(:/png/main/btn.png);
}

#generalIcon{
border-image: url(:/png/main/icon_general.png);

}

QLabel{
color: rgb(26, 95, 180);
font: 600 20pt &quot;Ubuntu&quot;;

}</string>
           </property>
           <widget class="QLabel" name="generalIcon">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>80</width>
              <height>80</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QLabel" name="generalText">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>140</y>
              <width>130</width>
              <height>36</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>常规</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="stepLadderBtn" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#stepLadderBtn{
border-image: url(:/png/main/btn.png);
}

#stepLadderIcon{
border-image: url(:/png/main/icon_stepladder.png);
}

QLabel{
color: rgb(26, 95, 180);
font: 600 20pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <widget class="QLabel" name="stepLadderIcon">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>80</width>
              <height>80</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QLabel" name="stepLadderText">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>140</y>
              <width>130</width>
              <height>36</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>阶梯</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="stepBtn" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#stepBtn{
border-image: url(:/png/main/btn.png);
}

#stepIcon{
border-image: url(:/png/main/icon_step.png);
}

QLabel{
color: rgb(26, 95, 180);
font: 600 20pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <widget class="QLabel" name="stepIcon">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>80</width>
              <height>80</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QLabel" name="stepText">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>140</y>
              <width>130</width>
              <height>36</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>步阶</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>80</number>
         </property>
         <property name="leftMargin">
          <number>240</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>240</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="gradientBtn" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#gradientBtn{
border-image: url(:/png/main/btn.png);
}

#gradientIcon{
border-image: url(:/png/main/icon_gradient.png);
}

QLabel{
color: rgb(26, 95, 180);
font: 600 20pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <widget class="QLabel" name="gradientIcon">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>80</width>
              <height>80</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QLabel" name="gradientText">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>140</y>
              <width>130</width>
              <height>36</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>渐变</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="throughBtn" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#throughBtn{
border-image: url(:/png/main/btn.png);
}

#throughIcon{
border-image: url(:/png/main/icon_cross.png);
}

QLabel{
color: rgb(26, 95, 180);
font: 600 20pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <widget class="QLabel" name="throughIcon">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>80</width>
              <height>80</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QLabel" name="throughText">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>140</y>
              <width>130</width>
              <height>36</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>低压穿越</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="setBtn" native="true">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>220</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#setBtn{
border-image: url(:/png/main/btn.png);
}

#setIcon{
border-image: url(:/png/main/icon_set.png);
}

QLabel{
color: rgb(26, 95, 180);
font: 600 20pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <widget class="QLabel" name="setIcon">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>80</width>
              <height>80</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QLabel" name="setText">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>140</y>
              <width>130</width>
              <height>36</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string>系统设置</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
         <item>
          <widget class="QLabel" name="versionLab">
           <property name="text">
            <string>软件版本: 0.003V</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="phoneLab">
           <property name="text">
            <string>联系方式: 1258465515</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
